cirrus/server/Dockerfile

```diff
@@ -1,6 +1,29 @@
+# Use official Python runtime as a parent image
 FROM python:3.11.2-slim-buster
+
+# Set working directory for the container
 WORKDIR /usr/src/cirrus
```

Comment:
Oh yeah we're gonna wanna make this into `/app/` to stay consistent with other service containers.

Reply:
good call, thank you :)